mapillary/models/api/entities.py

```diff
@@ -117,20 +116,22 @@ def fetch_map_feature(self, map_feature_id: int, fields: list = []):
         """
 
         # Getting the results through the client, and return after decoding
-        return self.client.get(
-            # Calling the endpoint with the parameters ...
-            Entities.get_map_feature(
-                # ... image_id, for the needed image ...
-                map_feature_id=map_feature_id,
-                # ... the fields passed in in ...
-                fields=fields
-                # ... only if the fields are not empty ...
-                if fields != []
-                # ... if they are, get all the fields as a list instead
-                else Entities.get_map_feature_fields(),
-            ),
-            # After retrieval of response, only get the content, decode to utf-8
-        ).content.decode("utf-8")
+        return ast.literal_eval(
```

Comment:
I'm worried about `ast.literal_eval` - this returns the `dict` as a `str` after running `eval` on the parameter post evaluation. Maybe we can introduce a `json.load` after this?

Reply:
When I was testing this, the output was `dict` not a `str`. Can you please elaborate on this? I was using the same convention you used when returning results from [EntityAdapter.fetch_image()](https://github.com/facebookincubator/mapillary-python-sdk/blob/de2dc7c3e9fdb85ac2351e1363704f7d2103eb37/mapillary/models/api/entities.py#L64)